### PR TITLE
Users/v imaga/workflows

### PR DESCRIPTION
**Pull request by Ibrahimmaga**
# Pull Request

## 📖 Description
Staring CI/CD workflows

### General

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@microsoft/fast-tooling-repository",
   "description": "A system of development tools, and utilities used à la carte or as a suite to build enterprise-grade websites and applications.",
-  "version": "0.1.0",
+  "version": "3.0.0",
```
Comment (janechu) [inline package.json:4]: Why was this updated?

File: .github/workflows/CI-Fast-Tooling [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: CI-Fast-Tooling
```
Comment (janechu) [inline .github/workflows/CI-Fast-Tooling:1]:
```suggestion
name: CI - FAST Tooling
```

File: .github/workflows/CI-Fast-Tooling [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+on:
+  push:
+    branches: [ release ]
```
Comment (janechu) [inline .github/workflows/CI-Fast-Tooling:5]: Reviewing https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration looks like the syntax for specifying branches should be:

```
branches:
  - release
```

File: .github/workflows/CI-Fast-Tooling [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  pull_request:
+    branches: [ release ]
```
Comment (janechu) [inline .github/workflows/CI-Fast-Tooling:7]: Same issue here as noted about branch syntax above.

File: .github/workflows/CI-Fast-Tooling [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```
Comment (janechu) [inline .github/workflows/CI-Fast-Tooling:15]: Is there a point to using a strategy matrix if there is only one option we are running the build against?

File: .github/workflows/CI-Fast-Tooling [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Build
+      run: |
+        npm install
```
Comment (janechu) [inline .github/workflows/CI-Fast-Tooling:27]: If this is the build gate shouldn't there be a testing step?

**Review by janechu — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Ibrahimmaga) [reply package.json:4]: This is a draft PR. Just for testing

**Comment by Ibrahimmaga** [comment]
This was created just for demo

**Pull request closed without merging**